.github/workflows/BuildPreCICE.yml

```diff
@@ -12,5 +12,21 @@ jobs:
          
       - run: julia -e 'println("Julia installed successfully!")'
       - uses: actions/checkout@v2
+      
+      - name: Cache preCICE
+        id: cache-libprecice
+        uses: actions/cache@v2
+        with:
+          path: libprecice2_2.2.0_focal.deb
```

Comment:
Lets use the latest release of preCICE [v2.3.0](https://github.com/precice/precice/releases/tag/v2.3.0) in this workflow